src/quacc/recipes/espresso/core.py

```diff
@@ -284,3 +284,59 @@ def post_processing_job(
         additional_fields={"name": "pp.x post-processing"},
         copy_files=prev_dir,
     )
+
+
+@job
+def non_scf_job(
+    atoms: Atoms,
+    prev_dir: str | Path | None = None, 
+    preset: str | None = "sssp_1.3.0_pbe_efficiency",
+    parallel_info: dict[str] | None = None,
+    test_run: bool = False,
```

Comment:
pw.x/ph.x/neb.x/cp.x jobs, but it is not needed on jobs like ase_relax_job for example,. because it does not make a lot of sense. To answer your question: the keyword will not always appear.